vendors/halo2/src/bn254_shplonk_prover.cc

```diff
@@ -27,20 +31,34 @@ rust::Box<G1JacobianPoint> DoCommit(
       reinterpret_cast<G1JacobianPoint*>(result));
 }
 
-std::vector<math::bn254::Fr> ReadFrs(uint8_t* vec_ptr, size_t i) {
+std::vector<math::bn254::Fr> ReadFrsToRemove(uint8_t* vec_ptr, size_t i) {
```

Comment:
Do we need this temporary change?

Reply:
This is coined from below reason. `ReadEvals` anyway should be renamed and `ReadFrs` are renamed correspondently.